test/benchmark/main.cc

```diff
@@ -0,0 +1,27 @@
+// NOLINT(namespace-envoy)
+// This is an Envoy driver for benchmarks.
+
+#include "benchmark/benchmark.h"
+
+#ifdef ENVOY_HANDLE_SIGNALS
+#include "common/signal/signal_action.h"
+#endif
+
+#include "absl/debugging/symbolize.h"
+
+// Boilerplate main(), which discovers benchmarks and runs them.
+int main(int argc, char** argv) {
+#ifndef __APPLE__
+  absl::InitializeSymbolizer(argv[0]);
+#endif
+#ifdef ENVOY_HANDLE_SIGNALS
+  // Enabled by default. Control with "bazel --define=signal_trace=disabled"
+  Envoy::SignalAction handle_sigs;
+#endif
+
```

Comment:
Are we doing enough initialization here to support general purpose Envoy code in benchmarks? E.g. if we want to enable logging while debugging a benchmark, would we want something like https://github.com/envoyproxy/envoy/blob/master/test/test_runner.cc#L108?

Reply:
We need to start from somewhere and iterate.  If you want me to add some log config lines, I can do that.

Reply:
I think making sure logging works would be great.